src/translations/translations.fr.js

```diff
@@ -45,16 +53,24 @@ const formatBytes = prepareNumberBytesFormatter(lang, 'o', '\u202f');
 const BYTES_SI_SEPARATOR = '\u202f';
 const formatBytesSi = prepareNumberUnitFormatter(lang, 'o', BYTES_SI_SEPARATOR);
 
+/**
+ * @param {number} value
+ * @return {string}
+ */
 function getUnit(value) {
   return formatBytesSi(value).split(BYTES_SI_SEPARATOR)[1];
 }
 
 // Shared logic between translations, is it a good idea?
-function formatFlavor(f) {
-  const cpu = `CPUs : ${f.cpus}`;
-  const shared = f.microservice ? ` (partagé)` : '';
-  const gpu = f.gpus > 0 ? `GPUs : ${f.gpus}` : '';
-  const mem = `RAM : ${formatBytes(f.mem * 1024 * 1024)}`;
+/**
+ * @param {Flavor} flavor
+ * @return {string}
+ */
+function formatFlavor(flavor) {
+  const cpu = `CPUs : ${flavor.cpus}`;
+  const shared = flavor.microservice ? ` (partagé)` : '';
+  const gpu = flavor.gpus > 0 ? `GPUs : ${flavor.gpus}` : '';
+  const mem = `RAM : ${formatBytes(flavor.mem * 1024 * 1024)}`;
   return [cpu + shared, gpu, mem].filter((a) => a).join('\n');
 }
 
```

Comment:
issue: I think we decided we would remove the `LANGUAGE` key as we no longer need it and it brings more problems than solutions

:arrow_right: this change is available in my proposition branch: https://github.com/CleverCloud/clever-components/tree/typecheck-i18n--hubert